content/hardware/06.nicla/boards/nicla-voice/tutorials/user-manual/content.md

```diff
@@ -439,7 +439,262 @@ The onboard magnetometer of the Nicla Voice can be used to determine the board's
+  Serial.println(z_acc);
+
+  // Print gyroscope data (expressed in degrees per second). 
+  Serial.print("x_gyr:");
+  Serial.print(x_gyr);
+  Serial.print(",");
+  Serial.print("y_gyr:");
+  Serial.print(y_gyr);
+  Serial.print(",");
+  Serial.print("z_gyr:");
+  Serial.println(z_gyr);
+
+  delay(1000);
+}
+```
+
+Here you can find a step-by-step explanation of the code:
+
+First, the necessary libraries are included: 
+
+- `NDP.h` and `BMI270_Init.h` for the Nicla Voice board's basic functions and the IMU control.
+- Macros are defined for checking the status of the IMU; these macros allow the sketch to detect and handle sensor errors. 
+
+Next, user functions `ledBlueOn()`, `ledGreenOn()`, and `ledRedBlink()` definition: 
+
+- These functions allow the onboard LEDs to flash specific colors to indicate different states: blue for a successful match, green for an event, and red to indicate an error.
+
+Next, in the `setup()` function:
+
+- The serial communication is initialized at a baud rate of 115200.
+- The Nicla Voice board is initialized, and the LDO is disabled.
+- Error and event handlers are initialized.
+- NDP processor is initialized; this process includes populating the external Flash memory of the board with the NDP processor internal microcontroller firmware (`mcu_fw_120_v91.synpkg`), the NDP processor internal DSP firmware (`dsp_firmware_v91.synpkg`), and the ML model (`ei_model.synpkg`). 
+- The BMI270 sensor is initialized; this includes a software reset, loading the sensor configuration, and setting it into normal power mode with the accelerometer and gyroscope operational. 
+
+Finally, in the `loop()` function:
+
+- Memory is allocated for the sensor data; data is then read from the sensor and stored in this allocated space.
+- Raw sensor data is then parsed and extracted into raw accelerometer and gyroscope data. This data is represented as 16-bit signed integers ranging from -32,768 to 32,767.
+- Raw sensor data is converted into understandable and standard unit measurements; for the accelerometer, data is converted to meters per second squared, and for the gyroscope, data is converted to degrees per second. 
+- Converted accelerometer and gyroscope data are printed to the Serial Monitor, allowing one to observe sensor data in real time.
+
+After uploading the example code, you should see accelerometer and gyroscope data on the IDE's Serial Monitor as shown below:
+
+![Nicla Voice onboard IMU data on the IDE's Serial Monitor](assets/user-manual-13.png)
+
+Let's use also the Arduino IDE Serial Plotter to test the example IMU sketch; let's visualize first only accelerometer data. To do this, **comment the gyroscope data output to the IDE's Serial monitor and upload the example sketch again**; then, open the IDE's Serial Plotter by navigating to **Tools -> Serial Plotter**. After a while, you should see a real-time graph showing data from the board's onboard accelerometer, as shown below (move the board):

```

Comment:
As a user I don't understand what and what I have to comment.

I suggest to improve the phrase and also add a small block code below with that part of the code where the user can be the part that is commented and the part that is uncommented

Reply:
You are right, I added the code section and changed a little bit the phrase.